javascript/javascript3/week3/shoppingCart/shoppingCrt.js

```diff
@@ -0,0 +1,101 @@
+const searchbar = document.getElementById("search")
+let ul = document.getElementById("productlist")
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    if (currency === "dollars") {
+      //one dollar = 6.28dkk
+      this.price = this.price / 6.28;
+      return this.price;
+    } else if (currency === "pound") {
+
+      this.price = this.price / 8.62;
+      return this.price;
+    } else if (currency === "euro") {
+
+      this.price = this.price / 7.43;
+      return this.price;
+    }
+
+  }
+}
+
+//shopping
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+  addProduct(product) {
+    this.products.push(product);
+  }
+  removeProduct(product) {
+    this.products = this.products.filter((item) => item.name !== product)
+    console.log(this.products);
+  }
+  searchProduct(productName) {
+    console.log(productName);
+    let searchresult = this.products.filter((items) => items.name === productName)
+    console.log(searchresult);
+    return searchresult;
+  }
+  getTotal() {
+    const total= this.products.map(item=>item.price).reduce((sum,price)=>sum+price,0)
+    console.log(total);
+    const h3 =document.createElement("h2");
```

Comment:
Here you are creating h2 tag and assigning to h3 which u have consider as variable name. 
We never use html tag as a variable name . Assign some other name.